addon/components/au-toaster.hbs

```diff
@@ -1,17 +1,45 @@
 {{#if this.toaster.toasts.length}}
   <div class="au-c-toaster {{this.position}}">
     {{#each this.toaster.toasts as |toast|}}
-      <AuAlert
-        @title={{toast.title}}
-        @skin={{toast.options.type}}
-        @icon={{toast.options.icon}}
-        @size="small"
-        @closable={{toast.options.closable}}
-      >
-        {{#if toast.message}}
-          <p>{{toast.message}}</p>
-        {{/if}}
-      </AuAlert>
+      {{#if toast.options.action}}
+        <AuAlert
+          @title={{toast.title}}
+          @skin={{toast.options.type}}
+          @icon={{toast.options.icon}}
+          @size="small"
+          @closable={{toast.options.closable}}
+          @isAToast={{true}}
```

Comment:
I'm not entirely sure this is the best (argument) name to differentiate the layout type within `AuAlert`?

Reply:
Can't we just pass a css class to the component?

Reply:
Oh, that is a better/simpler solution then mine 👍 I'll change it.